exercise3/check array nesting level.js

```diff
@@ -0,0 +1,38 @@
+function getNestingLevel(arr) {
+  let level = 0;
+
+  const checkDepth = (a, depth)=> {
+    if (Array.isArray(a)) {
+      level = Math.max(level, depth);
+      for (let item of a) {
+        checkDepth(item, depth + 1);
+      }
+    }
+  }
+
+  checkDepth(arr, 1);
+  return level;
+}
+
+function flattenArray(arr) {
+  let flat = [];
+
+  const flatten = a =>{
+    for (let item of a) {
```

Comment:
Check out why?
https://dev.to/edriso/demystifying-const-in-for-loops-traditional-vs-forof-3aif

```suggestion
    for (const item of a) {
```